pkg/accounting/tally/tally.go

```diff
@@ -103,75 +92,53 @@ func (t *tally) identifyActiveNodes(ctx context.Context) (err error) {
 				for _, p := range pieces {
 					nodeIDs = append(nodeIDs, p.NodeId)
 				}
-				online, err := t.onlineNodes(ctx, nodeIDs)
+				nodeData, err = t.calculate(ctx, pointer, nodeIDs)
 				if err != nil {
 					return Error.Wrap(err)
 				}
-				go t.tallyAtRestStorage(ctx, pointer, online, client)
 			}
 			return nil
 		},
 	)
-	return err
-}
-
-func (t *tally) onlineNodes(ctx context.Context, nodeIDs storj.NodeIDList) (online []*pb.Node, err error) {
-	responses, err := t.overlay.BulkLookup(ctx, pb.NodeIDsToLookupRequests(nodeIDs))
-	if err != nil {
-		return []*pb.Node{}, err
-	}
-	nodes := pb.LookupResponsesToNodes(responses)
-	for _, n := range nodes {
-		if n != nil {
-			online = append(online, n)
-		}
-	}
-	return online, nil
+	return t.updateRawTable(ctx, nodeData)
```

Comment:
It should update on every iteration, currently it's updating only for the last data.